methods/vtid.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "vtid",
+  "status": "registered",
+  "verifiableDataRegistry": "",
+  "contactName": "",
+  "contactEmail": "",
+  "contactWebsite": "",
```

Comment:
Please specify this information. At a minimum, you need to include a contact email address. Your security and privacy considerations section really needs to elaborate more on the considerations. You do not specify the syntax.

Reply:
Relevant information and syntax have been added，kindly review .